src/woo_publications/publications/api/filters.py

```diff
@@ -2,7 +2,38 @@
 
 from django_filters.rest_framework import FilterSet, filters
 
-from ..models import Publication
+from ..models import Document, Publication
+
+
+class DocumentFilterSet(FilterSet):
+    # TODO: change this filter to custom named filter with `@extend_schema_field(UUID)` once bug is fixed in drf-spectacular
```

Comment:
can you please create an issue for this problem at https://github.com/tfranzel/drf-spectacular/issues